### ISSUE TITLE
Unrendered @loadOnce directive on create/edit user view

### ISSUE THREAD

**Issue by joeycoonce** [Laravel-Backpack/PermissionManager#296]
# Bug report

### What I did:
Installed permissionmanager on fresh install of Laravel and Backpack

### What I expected to happen:
Blade directives rendered, not displayed

### What happened:
When creating or editing users, the text `@loadOnce('bpFieldInitChecklistDependencyElement') @endLoadOnce` is displayed on the lower left corner of page.
 
### What I've already tried to fix it:
Recreated issue with fresh install
Determined the code is lines 199-286 in /vendor/backpack/crud/src/resources/views/crud/fields/checklist_dependency.blade.php
Directly updated digitallyhappy/assets in permissionmanager via composer, no effect.

### Backpack, Laravel, PHP, DB version:
Laravel v9.3.1
PHP v8.0.14
Backpack CRUD v5.0.9
laravel-permission v5.5.1
permissionmanager v6.0.14

### Screenshot for reference:
![Screenshot 2022-03-05 6 32 29 AM](https://user-images.githubusercontent.com/1059971/156883593-e5d286c7-6e7a-4ea1-bc72-992114f5b742.png)

**Comment by pxpm** [comment]
Hey @joeycoonce  thanks for the report.

Could you confirm you did `php artisan view:clear` ?

We already fixed something similar to that, so I guess you just need to update your dependencies https://github.com/DigitallyHappy/assets/pull/11#pullrequestreview-878670128

Let me know if I am wrong. 

Thanks

**Comment by joeycoonce** [comment]
@pxpm I can confirm that I have run `php artisan view:clear` and that did not resolve the issue.
Since this issue is present on a fresh install as of this morning, I could use a little guidance as to which dependencies would need to be updated. I tries requiring `digitallyhappy/assets`,currently 2.0.3, in my project directly, as well as updating `backpack/crud` and `backpack/permissionmanager`, (and rerunning  `php artisan view:clear`), just to see if it would resolve the issue, whitch it did not.

**Comment by joeycoonce** [comment]
@pxpm I can confirm that I have run `php artisan view:clear` and that did not resolve the issue.
Since this issue is present on a fresh install as of this morning, I could use a little guidance as to which dependencies would need to be updated. I tried requiring `digitallyhappy/assets`,currently 2.0.3, in my project directly, as well as updating `backpack/crud` and `backpack/permissionmanager`, (and rerunning  `php artisan view:clear`), just to see if it would resolve the issue, whitch it did not.

**Comment by pxpm** [comment]
Indeed assets 2.0.3 is the one that fixed a similar issue. https://github.com/DigitallyHappy/assets/releases/tag/2.0.3

It's only happening with @loadOnce directive? Other directives work fine? You can for example use @hasRole from permission manager to test if it works?

**Comment by joeycoonce** [comment]
As I have only encountered the error within the implementation of the permissionmanager module, I haven't previously used these directives on the front end. Per your direction @pxpm, I tested using these blade directives on a frontend view and encountered a similar failure to render when using `@loadOnce`. `@hasRole` also failed, but `@hasrole` rendered and worked as expected (`@loadonce` still failed to render).

**Comment by joeycoonce** [comment]
As I have only encountered the error within the implementation of the permissionmanager module, I haven't previously used either of these directives directly. As requested, I tested using these blade directives on a frontend view and encountered a similar failure to render when using `@loadOnce`. `@hasRole` also failed, but `@hasrole` rendered and worked as expected (`@loadonce` still failed to render).
